web-console/knikubevirt/migrate-vm/migrate-vm.md

```diff
@@ -0,0 +1,19 @@
+# Migrate VM
+
+Clicking the action button next to any Virtual Machine in the List will present the user with the option to migrate the VM to another host.
+
+![Selecting a VM](img/1-0-list.png)
+
+A modal will be presented where the user can choose which Node the VM should be migrated to. 
+
+![Modal view](img/1-1-modal-info.png)
```

Comment:
This is a nitpick, but the "migrate" buttons have too much padding in the modals. Maybe we can update easily?